### PR TITLE
Travel Fund Request for @benjamingr at Collaborator Summit Spring 2018

### PR DESCRIPTION
**Pull request by benjamingr**
Hey,

I've never done this before so let me know if I'm doing this right.

Event: Collaborator Summit
Location: Berlin, Germany
Date: May 31 - Jun 3 2018
From: Tel Aviv, Israel
Amount: USD $400

I've never participated in a collaborator summit or Node.js foundation event outside of Israel before. I organize a lot of the Node.js related meetups in Israel and have been a core collaborator for a few years now.

I often find that I have communication barriers with people I want to collaborate with in the org and I think this would be a great opportunity to finally have all those discussions I've been saving up for years. 

In particular, I want to discuss better promise support in core, discuss API issues, collaborate with the rest of the module team and the moderation team whom I am a member of, discuss promises and postmortem debugging with the postmortem WG (I was unable to attend the Ottowa meetings).

$400 should be sufficient for covering a cheap flight and I'll try to stay over with a friend from Berlin to not have to take an expensive hotel :)

### CODE REVIEW

File: MEMBER_TRAVEL_FUND.md [hunk 1]
```diff
@@ -62,6 +62,6 @@ Tobias Nießen | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver, B
 Tiancheng Gu | NINA 2017 | Collaborator Summit | Vancouver, BC, CA | 3 Oct – 8 Oct 2017 | US$ 1200
 Ruben Bridgewater | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver, BC, CA | 4 Oct - 8 Oct 2017 | €1500
 Joyee Cheung | Diagnostics WG Summit | Attendance | Ottawa, ON, CA | 12 Feb - 13 Feb 2018 | US$ 1400
-
+Benjamin Gruenbaum| Collab Summit Spring 2018 | Attendence & Code & Learn Mentor | Berlin, Germany| May 31 - Jun 3 2018 | $400
```
Comment (oe) [inline MEMBER_TRAVEL_FUND.md]: some of these aren't properly spaced

File: MEMBER_TRAVEL_FUND.md [hunk 2]
```diff
@@ -62,6 +62,6 @@ Tobias Nießen | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver, B
 Tiancheng Gu | NINA 2017 | Collaborator Summit | Vancouver, BC, CA | 3 Oct – 8 Oct 2017 | US$ 1200
 Ruben Bridgewater | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver, BC, CA | 4 Oct - 8 Oct 2017 | €1500
 Joyee Cheung | Diagnostics WG Summit | Attendance | Ottawa, ON, CA | 12 Feb - 13 Feb 2018 | US$ 1400
-
+Benjamin Gruenbaum | Collab Summit Spring 2018 | Attendence & Code & Learn Mentor | Berlin, Germany | May 31 - Jun 3 2018 | $400
```
Comment (tniessen) [inline MEMBER_TRAVEL_FUND.md]: Tiny nit: `Attendence` → `Attendance`

Comment (benjamingr) [reply MEMBER_TRAVEL_FUND.md]: Good catch!

**Comment by benjamingr** [comment]
@mhdawson Thanks! I've been an individual member of the foundation ~ since io.js joined the Node.js foundation. I recall renewing it at least once using the /members link - in addition my membership is as an individual (that is, I am not and was never paid to work on Node.js core)

Is there any way to check that I'm still a member or I need to take an action to renew my membership?

**Comment by mhdawson** [comment]
@mrhinkle can you answer the question about how to check if you are still a member?

**Comment by joyeecheung** [comment]
@benjamingr Hi, there is a conflict in this document, can you rebase? There have been no objections in 18 days so I'll go ahead and merge this when the conflict is resolved.

**Comment by benjamingr** [comment]
Thanks @joyeecheung - conflicts fixed

**Comment by SofiEstevez** [comment]
Hi @benjamingr  ! I'm Sofi from This Dot team, writing on behalf of Node.js Foundation.

I don't see your report for reimbursement of this in the official md:
https://github.com/nodejs/admin/blob/master/TravelFunds/2018.md

(Apologies if you sent it already, it is possible that this is not updated)

We just wanted to let you know the last day to ask for this reimbursement is Jan 19th.
After that date, reimbursements for 2018 travels will no longer be available.

Thank you!

**Comment by benjamingr** [comment]
@SofiEstevez thanks (and thanks a ton for being on top of things!).

I sent an email on Oct 1 to travelapprovals@nodejs.org and got a response back: "Thank you Benjamin! We'll let you know if there are any issues!" from @ladyleet 

I was under the impression this is just taking time on the administrative side - let me know if anything is required from my side or if I can help.

Cheers.

**Pull request merged**